app/models/nic/interface.rb

```diff
@@ -74,9 +74,11 @@ def identifier_change
       old_value = self.identifier_was || ''
```

Comment:
The reason behind this is s390x machines issues dynamic MAC addresses on boot (so I'm told) to configure s390x NIC's via kickstart on the network line/s you need to use `--device=<device ID>` e.g `--device="enccw0.0.1234"` (s390x device names have period in). I ran into this problem where I couldn't add a device ID to a NIC without deleting and adding it again.

I wouldn't say your are changing a device type if you never specified it in the first place?

The  good news is to fully support s390x provisioning via kickstart it looks like we just need to make some template changes and a few changes to the host form validation.